.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+---
+defaults: &defaults
+  docker:
+    - image: canonicalwebteam/dev
+  working_directory: /srv
+
+version: 2
+jobs:
+  test-build:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: bundle install
+      - run:
+          name: Build the site resources
+          command: jekyll build
+      - run:
+          name: Run the site server
+          command: jekyll serve
+          background: true
+      - run:
+          name: Check site is accessible
+          command: sleep 1 && curl --head --fail --retry-delay 5 --retry 10  --retry-connrefused http://127.0.0.1:4000/practices/
+  run-prettier:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Run Prettier over all files
+          command: npx prettier --write **/*.*
```

Comment:
I don't think `npx` is available within our development image - could we instead use `node_modules/.bin/prettier`?

Also, I don't think prettier itself is installed, so you probably need a `package.json` with an `Install dependencies` step, with `yarn install`.

Thirdly, won't this command actually do the reformatting? I think what we really want here is just a CI check - as in we probably want `prettier --check`.